pages/common/lolcat.md

```diff
@@ -0,0 +1,19 @@
+# lolcat
+
+> Put a rainbow in everything you cat to console.
```

Comment:
cat within backticks ` `` `

Reply:
Will do that, but is this mentioned in the styleguide?
Or should it be mentioned in the styleguide (new PR)?

Reply:
It is not. Because it is a client dependent thing. The node client catches backticks and checks whether a command exists with the same name, and shows a "See also: cat" section. Other clients may not do that. 

Hence its a recommendation, not a style guide material.

Reply:
Now I understand.

I have put `cat` in backticks.